README.md

```diff
@@ -0,0 +1,61 @@
+ZenstruckAssetManifestBundle
+============================
+
+Installation
+------------
+
+Download:
+
+```
+composer require "zenstruck/asset-manifest-bundle:dev-master"
```

Comment:
Can you remove the dev-master and just have it as:

```
composer require zenstruck/asset-manifest-bundle
```

I will tag 1.0.0 when I merge this PR.